runtime/development/src/lib.rs

```diff
@@ -1354,14 +1354,15 @@ impl pallet_interest_accrual::Config for Runtime {
 
 impl pallet_connectors::Config for Runtime {
 	type AdminOrigin = EnsureRoot<AccountId>;
+	type AssetRegistry = OrmlAssetRegistry;
 	type Balance = Balance;
-	type CurrencyId = CurrencyId;
 	type Event = Event;
 	type Permission = Permissions;
 	type PoolInspect = PoolSystem;
 	type Rate = Rate;
 	type Time = Timestamp;
 	type Tokens = Tokens;
+	type TrancheToken = TrancheToken;
```

Comment:
`type TrancheCurrency = TrancheCurrency`

There exists a `TrancheCurrency` struct in the cfg-types that implements all you need already.